### ISSUE TITLE
Task column modification request

### ISSUE THREAD

**Issue by nosliwneb** [atheriel/org-clock-csv#3]
Would it be possible to include the header text from ancestors in the task column?

For an org file structured like this:

```
*Project 1
** Task1
    [clock data here]
** Task2
*Project 2
** Task1
** Task2
```

The current output is not so useful as the resulting CSV file doesn't contain any info to distinguish which project an individual clock event belongs to.

Rather than:

```
Task1,,,,,,
Task2,,,,,,
Task1,,,,,,
Task2,,,,,,
```

Maybe something like this:

```
Project1-Task1-SubTask1,,,,,,
Project1-Task2,,,,,,
Project2-Task1,,,,,,
Project2-Task2,,,,,,
```

**Comment by btljuice** [comment]
Hi,

The following pull request #6 includes a modification that adds headline ancestors to the task headline.

**Comment by atheriel** [comment]
This is resolved with #6. You can set the separator character to `-` if you wish.
